src/sentry/static/sentry/app/views/onboarding/project/platformpicker.jsx

```diff
@@ -87,24 +88,24 @@ class PlatformPicker extends React.Component {
                 type="text"
                 value={this.state.filter}
                 className="platform-filter"
-                label="Filter"
+                label={t('Filter')}
                 placeholder="Filter"
                 onChange={e => this.setState({filter: e.target.value})}
               />
             </div>
           </li>
-          {categoryList.map(categoryName => {
+          {allCategories.map(({id, name}) => {
```

Comment:
nice job detangling this logic  - I'm slightly afraid that a problem will show up when the strings get translated, but it looks like you handled the cases correctly as far as i can see